linz.js

```diff
@@ -468,9 +474,15 @@ Linz.prototype.bootstrapExpress = function (cb) {
 		this.app.use(expressSession({ secret: this.get('cookie secret') }));
 	}
 
-	// initialize passport
-	this.app.use(this.get('admin path'), passport.initialize());
-	this.app.use(this.get('admin path'), passport.session());
+	// setup connect-flash
+	this.app.use(flash());
+
+	// configure passport
+	libPassport(this.passport);
```

Comment:
How does this works when there's a custom passport already configured with `serializeUser` and `deserializeUser` (like in the rascamp example)?

It looks like the default function `serializeUser` and `deserializeUser` in `lib/passport.js` are ignored?